okta/users.go

```diff
@@ -90,34 +106,49 @@ func resourceUserRead(d *schema.ResourceData, m interface{}) error {
 
 	userList, _, err := client.Users.GetByID(d.Get("email").(string))
 	if err != nil {
-		// if the user does not exist in okta, delete from terraform
-		// results in a terraform plan to create the user
+		// if the user does not exist in okta, delete from terraform config
```

Comment:
minor but I think you mean `terraform state` not `terraform config`

Reply:
:+1: